src/info/langs/mod.rs

```diff
@@ -31,23 +31,13 @@ fn get_language_distribution(languages: &tokei::Languages) -> Option<HashMap<Lan
     let mut language_distribution = HashMap::new();
 
     for (language_name, language) in languages.iter() {
-        let mut code = language.code;
+        let mut loc = language::loc(language_name, language);
 
-        let has_children = !language.children.is_empty();
-
-        if has_children {
-            for reports in language.children.values() {
-                for stats in reports.iter().map(|r| r.stats.summarise()) {
```

Comment:
Looking at tokei's documentation for `summarise `:

>     /// Creates a new `CodeStats` from an existing one with all of the child
>     /// blobs merged.

If I'm not mistaken this method already handles the recursion for use 🤔

Reply:
Oh :laughing: I was so focused on the struct fields I missed that. Thanks for checking!